doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -82,6 +82,83 @@ pg = placement_group(
 ray.get(pg.ready())
 ```
 
+### TPU scheduling utility library
```

Comment:
add a note that this library is alpha status and subject to changes in future versions